### DISCUSSION TITLE
Unique 模型 ，flink写入时需要执行update table set 如何操作

### DISCUSSION THREAD

**Discussion by HK2009** [apache/doris#14022] [A - General / Q&A]
DataStream方式写入，有insert与update2种方式，官网的demo中只是insert的操作，试问如何执行update操作呢

**Comment by JNSimba** [comment]
封装RowData，将RowKind设置为UPDATE即可

**Reply by HK2009** [reply]
此时如何设置where条件呢？

**Reply by HK2009** [reply]
这个方法设置为RowKind.UPDATE_AFTER，但是并无法设置where条件
